src/sentry/shared_integrations/client/proxy.py

```diff
@@ -129,7 +129,7 @@ def __init__(
         # If the request flow for processing a Webhook outbox message is between the RegionSiloClient and the
         # IntegrationProxyClient, then the IntegrationProxyClient will need to have a smaller timeout value.
         # Otherwise, the RegionSiloClient will timeout before it can receive a response from the IntegrationProxyClient.
-        self.timeout = 20
+        self.timeout = 10
```

Comment:
We should keep an eye on the number of timeouts as this could result in valid traffic hitting timeouts too.